### PR TITLE
Add Pallas's cat (Otocolobus manul) :3

### PR DESCRIPTION
**Pull request by Sightem**
I was looking at the selection combo box and there was no manul option?! how could this be?? this pull request should fix this issue

### CODE REVIEW

**Comment by Sightem** [comment]
previews:

manulxmasmanulbgmanulversary

**Comment by namedata** [comment]
yes please do this

**Comment by slendidev** [comment]
cute gato

**Comment by glowiak** [comment]
Well seems like everyone is adding their own cats now. Why can't everyone just don't use the cat as I do?

File: launcher/ui/MainWindow.cpp [hunk 1]
```diff
@@ -1606,6 +1606,9 @@ void MainWindow::setCatBackground(bool enabled)
 
         QString cat = "default";
         QString catStyleOpt = APPLICATION->settings()->get("CatStyle").toString();
+        
+        if(catStyleOpt == "Manul")
+            cat = "manul";
         if(catStyleOpt == "Floppa")
             cat = "floppa";
         else if(catStyleOpt == "Jinx")
```
Comment (LennyMcLennington) [inline launcher/ui/MainWindow.cpp]:
```suggestion
        
        if(catStyleOpt == "Manul")
            cat = "manul";
        else if(catStyleOpt == "Floppa")
            cat = "floppa";
        else if(catStyleOpt == "Jinx")
```

**Review by LennyMcLennington — CHANGES_REQUESTED** [review]

**Comment by Sightem** [comment]
fixed the commit sign offs

**Comment by Gentleman2292** [comment]
MANUL

**Review by Sneedplex — APPROVED** [review]
cat

**Pull request merged**